NEWS

```diff
@@ -15,12 +15,12 @@ Current git version
     - +monitors.by-name.+ is valid
     - +monitors..by-name.+ is *invalid*
 
-  * Many boolean style settings were formely of type int. Now, these are
+  * Many boolean style settings were formerly of type int. Now, these are
     boolean settings.
   * The 'toggle' command only works for boolean settings. For the former
-    behaviour for int-typed settings, use 'cycle_value'.
+    behavior for int-typed settings, use 'cycle_value'.
```

Comment:
Not sure if you prefer American or British ;)

Reply:
Normally British, but I think American English is more common for coding, so let's stick to American :)

Reply:
Normally British, but I think American English is more common for coding, so let's stick to American :)